internal/bots/slack/slack.go

```diff
@@ -0,0 +1,105 @@
+package slack
+
+import (
+	"context"
+	"errors"
+	"net/http"
```

Comment:
如果接受了 Listen and Serve 模式的修改的话记得也改一下 import

```suggestion
	"net" 
	"net/http"
```